libraries/AP_Airspeed/AP_Airspeed_I2C.cpp

```diff
@@ -80,7 +80,7 @@ void AP_Airspeed_I2C::_collect()
 
     _measurement_started_ms = 0;
 
-    if (_dev->transfer(nullptr, 0, data, sizeof(data))) {
+    if (!_dev->read_registers(0, data, sizeof(data))) {
```

Comment:
this is still puzzling me.  It doesn't make much sense according to the datasheet:
http://www.meas-spec.com/downloads/Interfacing_to_MEAS_Digital_Pressure_Modules.pdf

I think that one thing that may be happening here is that here it triggers a new conversion, _measure() is actually doing nothing and you always read the sample from the previous iteration.

Maybe just changing the _measure() to do `transfer(&val, 1, nullptr, 0)`, with val == 0. And here we do `transfer(nullptr, 0, &data, sizeof(data))`